lib/driver.js

```diff
@@ -771,12 +769,63 @@ class XCUITestDriver extends BaseDriver {
     }
     log.debug(`Setting initial orientation to '${orientation}'`);
     try {
-      await this.proxyCommand('/orientation', 'POST', {orientation});
+      await this._executeCommandWithTimeout(B.promisify(this.proxyCommand), 'setScreenOrientation', '/orientation', 'POST', {orientation});
       this.opts.curOrientation = orientation;
     } catch (err) {
       log.warn(`Setting initial orientation failed with: ${err}`);
     }
   }
+
+  _normalizeCommandTimeouts (value) {
```

Comment:
Since this does not need to be on the driver object, I'd much prefer it to be in `utils.js` or some other place like that.